Remove periodic commit (#1454)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc

  Alternatively, `LOAD CSV` also supports accessing CSV files via _HTTPS_, _HTTP_, and _FTP_.
* `LOAD CSV` supports resources compressed with _gzip_ and _Deflate_. Additionally `LOAD CSV` supports locally stored CSV files compressed with _ZIP_.
* `LOAD CSV` will follow _HTTP_ redirects but for security reasons it will not follow redirects that changes the protocol, for example if the redirect is going from _HTTPS_ to _HTTP_.

> * `LOAD CSV` is often used in conjunction with the query clause `CALL +++{ ... }+++ TRANSACTIONS`. More information on this may be found in <<subquery-call-in-transactions, CALL +++{ ... }+++ IN TRANSACTIONS>>.

Reviewer: ```suggestion
* `LOAD CSV` is often used in conjunction with the query clause `CALL +++{ ... }+++ TRANSACTIONS`. For more information, see <<subquery-call-in-transactions, CALL +++{ ... }+++ IN TRANSACTIONS>>.
```

include::importing-large-amounts-of-data.asciidoc[]

> include::setting-the-rate-of-periodic-commits.asciidoc[]

Reviewer 2: Is this just gone or should we import a replacement here?

Author: Oh, nice catch. This should be replaced, thank you.
